woocommerce/woocommerce-ios
# Jetpack CP: track `feature_jetpack_benefits_banner` and `jetpack_install_button_tapped` events

Part of #5402 


### Description


This PR tracks two events for JCP sites: `feature_jetpack_benefits_banner` and `jetpack_install_button_tapped`:

Event | Properties
-- | --
feature_jetpack_benefits_banner | action: "shown" \| "dismissed" \| "tapped"
jetpack_install_button_tapped | source: "settings"\|"benefits_modal"

### Testing instructions


Prerequisites: 
1. the Jetpack benefits banner hasn't been dismissed within 5 calendar days (you can log out and in again to reset)
2. the account is connected to at least a JCP site (setup tips in p1636091588189400-slack-C6H8C3G23)

- Launch the app, log in to a JCP site if needed --> a Jetpack benefits banner should appear at the bottom of My store tab. in the console, there should be an event `🔵 Tracked feature_jetpack_benefits_banner, properties: [AnyHashable("action"): "shown", ...]`
- Tap on the Jetpack banner --> in the console, there should be an event `🔵 Tracked feature_jetpack_benefits_banner, properties: [AnyHashable("action"): "tapped", ...]`
- Tap "Install Jetpack" and quickly dismiss the modal to stop installation --> in the console, there should be an event `🔵 Tracked jetpack_install_button_tapped, properties: [AnyHashable("source"): "benefits_modal", ...]`
- Tap "X" on the Jetpack banner --> in the console, there should be an event `🔵 Tracked feature_jetpack_benefits_banner, properties: [AnyHashable("action"): "dismissed", ...]`
- Go to Settings
- Tap "Install Jetpack" --> in the console, there should be an event `🔵 Tracked jetpack_install_button_tapped, properties: [AnyHashable("source"): "settings", ...]`

---
